src/app/registration/contact-form/contact-form.component.html

```diff
@@ -19,13 +19,21 @@ <h2>
             id="lastName">
     </div>
     <div class="form-group">
-        <label for="email">Email address</label>
+        <label for="email">Email Address</label>
         <input type="email"
             class="form-control"
             formControlName="email"
             id="email"
             placeholder="name@example.com">
     </div>
+    <div class="form-group">
+        <label for="phone">Phone Number (Optional)</label>
+        <input type="number"
```

Comment:
type "number" allows negative numbers and 'e' (exponent)

Reply:
de3434c